vach-cli/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "vach-cli"
-version = "0.2.2"
+version = "0.3.0"
```

Comment:
it might be a good idea to keep this version running alongside with the vach version, or is that a bad idea?

Reply:
Version 0.3.0 is the first spec-compliant version, it marks a new leaf really. Plus vach changed a bit